lib/components/Alert.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { PropTypes } from 'react';
+
+/**
+ * Provides one of four standard alert types.
+ * An alert consists of a box with a light background color,
+ * an icon, a headline, and a short explaination.
+ *
+ * The alert type determines the background color and icon displayed
+ * 
+ * Alert types are 'success', 'warning', 'error', 'info'
+ * The type defaults to 'info' if no type is sent via props
+ *
+ * If the type is 'error' and no role is specified, role defaults to 'alert'
+ *
+ * @param {string} type  Sets the alert type
+ * @param {string, required} title Text for the headline
+ * @param {ndoe, required} body  Text for the description
+ * @param {string} role  ARIA role type
+ */
+export default function Alert ({type, title, body, role}) {
+  if ( type === 'error' && !role) {
+    role = 'alert';
```

Comment:
This was me trying to cover all the bases by having it as a non-required option with a minimal amount of automation. 

So the only two options I know about are `alert` and `alertdialog`.  

`alert` is considered intrusive (i.e. screen readers may interrupt current output to read the alert) so it should only be used for important time-sensitive messages (like an invalid value on a form field or login session about to expire). Me automatically adding it to error alerts might be a questionable decision. But certainly seems reasonable to think there will be instances where the role is needed/required and those will be on a case-by-case basis.

`alertdialog` is probably an edge case because it requires that the alert be interactive — i.e. have at least one focusable control that automatically gets focus when the alertdialog appears. Given how the 18F alerts are designed, it doesn't seem super likely. But yet they felt the need to put the option in their docs.